exercises/exercise3.js

```diff
@@ -1,12 +1,419 @@
+    irregularShape: "yes",
+    color: "dark",
+    url:"https://upload.wikimedia.org/wikipedia/commons/f/f5/actephila_lindleyi_flockton.jpg",
+    imageCredit: "margaret flocton"
+  },
+  {
+    commonName: "chain fruit",
+    latinName: "alyxia ruscifolia",
+    indigenousName: " ",
+    locations: "dapto ck., mt. nebo, wattle ck., goondarrin ck.",
+    endangered: "yes",
+    info:
+      "restricted to rainforest in the mt. nebo to mt. kembla/ upper cordeaux area. population sizes vary but are typically small and very localised e.g. mt. nebo 2plants only. - by anders bofeldt",
+    type: "leaves",
+    petals: "hairy",
+    petalShape: "pointy, wavy",
+    irregularShape: "yes",
+    color: "dark",
+    url:"https://upload.wikimedia.org/wikipedia/commons/5/56/alyxia_ruscifolia_2c.jpg",
+    imageCredit: "consultaplantas"
+  }
+];
+
 
 module.exports.botScripts = [
   {
-    label: 'An Example Script',
-    prompt: 'Good Evening',
+    label: 'Hi script',
+    prompt: 'hi',
     handler: function () {
-      return 'Good Evening to You!';
+      return 'Hi there!';
     },
+    isListening: true,
+    isCaseSensitive: false,
   },
-];
+  {
+    label: 'Hello script',
+    prompt: 'Hello',
+    handler: function () {
+      return 'Hello human';
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Asking how you are script',
+    prompt: 'How are you?',
+    handler: function () {
+      return 'Can\'t complain. What about yourself?';
+    },
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  // {
+  //   label: 'Responding to how are you script',
+  //   prompt: 'I\'m #{word}, thanks',
+  //   handler: function (params) {
+  //     if (params.word === "good" || params.word === "great" 
+  //       || params.word === "ok" || params.word === "okay" 
+  //       || params.word === "stunning") {
+  //       return 'Good for you!';
+  //     } else {
+  //       return 'Why so ' + params.word + '?';
+
+  //     }
+  //   },
+  //   isReply: true,
+  //   isListening: true,
+  //   isCaseSensitive: false,
+  // },
+  {
+    label: 'Responding to how are you script',
+    prompt: 'I\'m #{word}',
+    handler: function (params) {
+      if (params.word === 'good' || params.word === 'great'
+        || params.word === 'ok' || params.word === 'okay'
+        || params.word === 'stunning') {
+        return 'Good for you!';
+      } else {
+        return 'Why so ' + params.word + '?';
+
+      }
+    },
+    // isReply: true,
+    isListening: true,
+    isCaseSensitive: false,
+  },
+  {
+    label: 'Responding to why so script 0',
+    prompt: 'because I don\'t understand how you work',
+    handler: function () {
+      return 'Okay, easy. Let me explain...' 
+      + '\nI\'m a Native Australian Plant IDing bot.'
+      + '\nI\'m here to help you discover plant species in the area.';
+    },
+    // isReply: true, --------------- If I have this set to true, terminal throws 'Shell: ...' before it's response?
```

Comment:
The 'isReply: true' error may have to do with your return on this one, but it is not completely necessary anyways, so it is okay to leave out
Also, this one made me chuckle!